src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -15,6 +14,7 @@ import {
   UploadParameters,
 } from 'src/libs/ajax/data-table-providers/DataTableProvider';
 import { LeoAppStatus, ListAppItem } from 'src/libs/ajax/leonardo/models/app-models';
+import { WorkspaceData } from 'src/libs/ajax/WorkspaceDataService';
 import { Capabilities, Capability } from 'src/libs/ajax/WorkspaceDataService';
```

Comment:
```suggestion
import { Capabilities, Capability, WorkspaceData } from 'src/libs/ajax/WorkspaceDataService';
```
You are importing WorkspaceDataService multiple times, recommend just adding WorkspaceData to the existing import.

Reply:
fixed